docs/guides/admin/docs/configuration/security.jwt.md

```diff
@@ -1,18 +1,107 @@
 Configuration for JWT-based Authentication and Authorization
 ============================================================
 
-This page describes how to configure Opencast to enable authentication and authorization based on
-[JSON Web Tokens (JWTs)](https://datatracker.ietf.org/doc/html/rfc7519). With this feature, a login-mechanism based
-on the [OpenID Connect (OIDC)](https://openid.net/connect/) protocol can be configured, since OIDC uses JWTs.
+This page describes how to configure Opencast to enable authentication and/or authorization based on
+[JSON Web Tokens (JWTs)](https://datatracker.ietf.org/doc/html/rfc7519). JWTs can be used in various ways:
+
+- With static file authorization (particularly useful with external tools).
+- External apps can send users to Opencast (e.g. for Studio or Editor) with a session that has certain roles.
+- A login-mechanism based on the [OpenID Connect (OIDC)](https://openid.net/connect/) protocol can be configured.
+
+*Note*: Some of these use cases might not be perfectly supported by Opencast yet, but are possible in principle.
 
 Prerequisites
 -------------
 
-This guide assumes that a JWT provider is already setup and Opencast receives the JWT within an HTTP request header,
-or in a request parameter (i.e. a query parameter for `GET`-requests, and a form parameter for `POST`-requests).
+Opencast does not generate JWTs, but reads them in incoming requests. This guide assumes that you already have a JWT provider, i.e. another services that generates JWTs for consumption in Opencast. This could be [Tobira](https://elan-ev.github.io/tobira/setup/auth/jwt), an LMS, an OIDC provider, or something else. These generally have their own setup guides. The JWT has to be passed to Opencast via HTTP header or via request parameter (i.e. a query parameter for `GET`-requests, and a form parameter for `POST`-requests).
+
 In order to integrate Opencast with an OIDC provider you could use the
 [oauth2-proxy](https://github.com/oauth2-proxy/oauth2-proxy).
 
+Standard OC Schema for JWTs
+---------------------------
+
+After the signature verification, a JWT is just a bunch of "claims" (JSON fields). How to interpret these claims is completely up to the communicating parties. Protocols like OIDC define exactly which claims are to be used and how. For other cases where external Opencast apps (like Tobira or an LMS) need to communicate with Opencast, this section defines a standard schema to follow. You can configure Opencast differently, but this aims at standardizing JWT usage across Opencast-related applications and offer an out-of-the-box solution.
```

Comment:
> How to interpret these claims is completely up to the communicating parties.

We do have RFCs that specify constraints and give definitions for certain claims. I would suggest removing or altering this sentence.